srcts/src/bindings/input/selectInput.ts

```
@@ -207,7 +222,10 @@ class SelectInputBinding extends InputBinding {
   initialize(el: SelectHTMLElement): void {
     this._selectize(el);
   }
-  protected _selectize(el: SelectHTMLElement, update = false): SelectizeInfo {
+  protected _selectize(
+    el: SelectHTMLElement,
+    update = false
+  ): SelectizeInfo | undefined {
     if (!$.fn.selectize) return undefined;
```

Let's have this throw instead, so the return type can not include undefined